common/display/src/test/scala/weco/catalogue/display_model/models/DisplayWorkTest.scala

```diff
@@ -462,6 +462,81 @@ class DisplayWorkTest
     }
   }
 
+  describe("works in a series") {
+
+    it("includes a series in partOf") {
+      val work = indexedWork(
+        relations = Relations(
+          ancestors = List(Relation("Series A")),
+          children = List(),
+          siblingsPreceding = List(),
+          siblingsSucceeding = List()
+        )
+      )
+      val displayWork =
+        DisplayWork(work, WorksIncludes(WorkInclude.PartOf))
+      displayWork.partOf.isEmpty shouldBe false
+      val partOf = displayWork.partOf.get
+      partOf.map(_.id) shouldBe List(None)
+      partOf.map(_.title) shouldBe List("Series A")
+
+    }
+
+    it("can include multiple series") {
+      val work = indexedWork(
+        relations = Relations(
+          ancestors = List(Relation("Series A"), Relation("Series B")),
+          children = List(),
+          siblingsPreceding = List(),
+          siblingsSucceeding = List()
+        )
+      )
+      val displayWork =
+        DisplayWork(work, WorksIncludes(WorkInclude.PartOf))
+      displayWork.partOf.isEmpty shouldBe false
+      val partOf = displayWork.partOf.get
+      partOf.flatMap(_.title) shouldBe List("Series B", "Series A")
+      // series partOfs have no id.
+      partOf.flatMap(_.id) shouldBe List()
```

Comment:
Any reason you've switched from `.map` in the first test to `.flatMap` here?

Reply:
The first test is wrong (and fails). It's flatMap because the values are now optional.